src/main/java/com/sprint/mission/discodeit/dto/entity/Channel.java

```diff
@@ -0,0 +1,38 @@
+package com.sprint.mission.discodeit.dto.entity;
+
+import jakarta.validation.constraints.NotEmpty;
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+public class Channel extends BaseEntity implements Serializable {
+    @NotEmpty
+    private String channelName;
+    private String description;
+    private ChannelGroup channelGroup;
+    private UUID madeUserId;
+
+    public Channel(String channelName, String description, ChannelGroup channelGroup) {
+
+        this.channelGroup = channelGroup;
+        this.channelName = channelName;
+        this.description = description;
```

Comment:
사소한 부분이지만 인자 순서대로 내부도 작성하면 가독성이 더 좋을 것 같아요.   
```suggestion
    public Channel(String channelName, String description, ChannelGroup channelGroup) {

        this.channelName = channelName;
        this.description = description;
        this.channelGroup = channelGroup;
```